alpaca/broker/requests.py

```diff
@@ -987,8 +987,8 @@ class GetEventsRequest(NonEmptyRequest):
     id: Optional[str] = None
     since: Optional[Union[date, str]] = None
     until: Optional[Union[date, str]] = None
-    since_id: Optional[int] = None
-    until_id: Optional[int] = None
+    since_id: Optional[Union[int, str]] = None
+    until_id: Optional[Union[int, str]] = None
```

Comment:
Thank you for this PR. I have commented similar to the other PR: https://github.com/alpacahq/alpaca-py/pull/588#issuecomment-2816492625 .
I feel this change might make us hard to remove a method calling deprecated API endpoint. Therefore, prefer to have separate set of method/request/response models instead of making existing one compatible. It is hard to migrate to a new version seamlessly. Therefore, prefer to provide a chance to notice of deprecated method/request/response models for the upcoming removing of them.